commons/css/skolar.css

```diff
@@ -1,20 +1,20 @@
 @font-face {
   font-family: "Skolar";
-  src: url("http://jktauber.com/site_media/static/fonts/SkolarPEWeb-Regular.woff") format("woff");
+  src: url("https://jktauber.com/site_media/static/fonts/SkolarPEWeb-Regular.woff") format("woff");
```

Comment:
I thought it was still worth doing even if not a valid cert. It certainly doesn't work with it as `http`, though, so not sure what else to do.